src/test/java/com/ays/assignment/controller/AssignmentSystemTest.java

```diff
@@ -94,6 +97,57 @@ void givenValidAssignmentSaveRequest_whenUserUnauthorizedForSaving_thenReturnAcc
 
     }
 
+    @Test
+    void givenNothing_whenUserAssignmentFound_thenReturnAssignmentUserResponse() throws Exception {
```

Comment:
`givenNothing_` yerine hiç yazmayabiliriz doğrudan `when...` ile başlar

Reply:
İlgili test class'ındaki method isimlerini düzelttim. İsimleri değiştirince test order değiştiği için verileri farlı sırada değiştiriyordu. Bu yüzden testlerde veriden kaynaklı hata oluşuyordu. `@Order` anotasyonu ile doğru sırada veriyi güncellemelerini sağladım.